pandas/core/base.py

```diff
@@ -946,32 +955,32 @@ def argmax(self, axis=None, skipna=True, *args, **kwargs):
         Returns
         -------
         int
-            Row position of the maximum values.
+            Row position of the {operation} values.
 
         See Also
         --------
-        numpy.ndarray.argmax : Equivalent method for numpy arrays.
-        Series.argmin : Similar method, but returning the minimum.
+        numpy.ndarray.{klass} : Equivalent method for numpy arrays.
+        Series.{oppose_klass} : Similar method, but returning the {oppose_operation}.
```

Comment:
could reduce the parameter count using min and max for oppose
```suggestion
        Series.arg{oppose} : Similar method, but returning the {oppose}imum.
```